### PR TITLE
DashboardScene: Adjust select options selectors in scenes e2e tests

### PR DESCRIPTION
**Pull request by Sergej-Vlasov**
Adjust select options selectors in e2e tests after https://github.com/grafana/grafana/pull/88118

Depends on: https://github.com/grafana/scenes/pull/770

### CODE REVIEW

File: e2e/scenes/dashboard-suite/dashboard-time-zone.spec.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ describe('Dashboard time zone support', () => {
         e2e.components.Select.input().should('be.visible').click();
       });
 
-    e2e.components.Select.option().should('be.visible').contains(toTimeZone).click();
+    cy.get("[aria-label='Select option']").should('be.visible').contains(toTimeZone).click();
```
Comment (ivanortegaalba) [inline e2e/scenes/dashboard-suite/dashboard-time-zone.spec.ts:60]: Looking at the referenced PR, [they use data-testid](https://github.com/grafana/grafana/pull/88118/files#diff-6d85d22d3e9b3a1961ecc27157632a0096b1732e084e86f1b498774e8c84a483R110) to refer to options. Why are we moving back here to aria-label?

Comment (Sergej-Vlasov) [reply e2e/scenes/dashboard-suite/dashboard-time-zone.spec.ts:60]: @ivanortegaalba I already introduced `data-testid` on [that component in scenes](https://github.com/grafana/scenes/blob/e718252f7b153fd1a3339165dea60042fc08e363/packages/scenes/src/variables/components/VariableValueSelect.tsx#L180) but it includes actual value for better cypress targeting. `aria-label` is only used to get total visible number of select options.

two ways forward:
1. leave as is with `aria-label`
2. move my custom `data-testid` in [scenes](https://github.com/grafana/scenes/blob/e718252f7b153fd1a3339165dea60042fc08e363/packages/scenes/src/variables/components/VariableValueSelect.tsx#L180) a level down and introduce `data-testid` on parent as in #88118


Probably best to do option 2 to keep everything consistent

Comment (ivanortegaalba) [reply e2e/scenes/dashboard-suite/dashboard-time-zone.spec.ts:60]: I'd go always for data-testid. We're deprecating the usage of aria-label for targeting elements in favor of data-testid :)

**Review by ivanortegaalba — APPROVED** [review]
🙌 Thanks for making the effort of solving it from Scenes!! 🚀

**Comment by Sergej-Vlasov** [comment]
closing because this PR updates scenes version and its already done elsewhere

**Pull request closed without merging**